### PR TITLE
Proof LATITUDE_LONGITUDE_SUBROUTINES #223

### PR DESCRIPTION
**Pull request by dwiyatci**
🕵️ Proofread #223.

### CODE REVIEW

File: Comanche055/LATITUDE_LONGITUDE_SUBROUTINES.agc [hunk 1]
```diff
@@ -31,20 +31,21 @@
...
-#	R-TO-RP, ARCTAN, SETGAMMA, SETRE
 #
-# ERASABLE INIT. REQ.
-#	AXO, -AYO, AZO, TEPHEM (SET AT LAUNCH TIME)
-#	ALPHAV = POSITION VECTOR METERS B-29
-#	MPAC -- TIME (CSECS B-28)
-#	ERADFLAG =1, TO COMPUTE EARTH RADIUS, =0 FOR FIXED EARTH RADIUS
-#	LUNAFLAG=0 FOR EARTH, 1 FOR MOON
+#	R-TO-RP,ARCTAN,SETGAMMA,SETRE
+# ERASABLE INIT.  REQ.
```
Comment (tjchambers) [inline Comanche055/LATITUDE_LONGITUDE_SUBROUTINES.agc]:
```suggestion
# ERASABLE INIT. REQ.
```

File: Comanche055/LATITUDE_LONGITUDE_SUBROUTINES.agc [hunk 2]
```diff
@@ -100,48 +101,46 @@ CALLRTRP	CALL
...
 			0D
-		STCALL	6D		#     6-7D = TIME FOR RP-TO-R
-			SETGAMMA	#    GAMMA = B2/A2 FOR EARTH, 1 FOR MOON B-1
-		DLOAD	SIN		#           COS(LONG)COS(LAT) IN MPAC
-			LAT		# UNIT  RP = SIN(LONG)COS(LAT)    2-3D
-		DMPR	PDDL		# PD 2      GAMMA*SIN(LAT)       0-1D
+		STCALL	6D		# 6-7D= TIME FOR RP-TO-R
+			SETGAMMA	# GAMMA=B2/A2 FOR EARTH,1 FOR MOON B-1
+		DLOAD	SIN		# 		COS(LONG)COS(LAT) IN MPAC
+			LAT		#    UNIT  RP = SIN(LONG)COS(LAT)    2-3D
```
Comment (tjchambers) [inline Comanche055/LATITUDE_LONGITUDE_SUBROUTINES.agc]:
```suggestion
			LAT		#    UNIT RP= SIN(LONG)COS(LAT)    2-3D
```

File: Comanche055/LATITUDE_LONGITUDE_SUBROUTINES.agc [hunk 3]
```diff
@@ -179,10 +179,10 @@ GETERAD		DLOAD	DSQ
 		STORE	ERADM
 			RVQ
 
-# THE FOLLOWING CONSTANTS WERE COMPUTED WITH A=6378166, B=6356784 METERS
-# B2XSC = B**2 SCALED B-51
-# B2/A2 = B**2/A**2 SCALED B-1
-# EE = (1-B**2/A**2) SCALED B-0
+# THE FOLLOWING CONSTANTS WERE COMPUTED WITH A=6378166,B=6356784 METERS
+# B2XSC= B**2 SCALED B-51
+# B2/A2= B**2/A**2  SCALED B-1
```
Comment (tjchambers) [inline Comanche055/LATITUDE_LONGITUDE_SUBROUTINES.agc]:
```suggestion
# B2/A2= B**2/A**2 SCALED B-1
```

**Comment by tjchambers** [comment]
If you value my reviews I am happy to continue. I have not finished this one and will not have time for 24 hours or so. Just an FYI.

**Comment by dwiyatci** [comment]
@tjchambers Sure thing, I value yours. I really do. Take your time - I think I'm not in a rush (or are we? 😃). Thank you! 🙇🏻🙇🏻

**Comment by wopian** [comment]
@tjchambers thanks! More eyes double checking against the scans is always better 🎉

**Pull request merged**